src/main/java/appeng/client/texture/FlippableIcon.java

```diff
@@ -36,87 +38,87 @@ public FlippableIcon( final IIcon o )
 			throw new IllegalArgumentException( "Cannot create a wrapper icon with a null icon." );
 		}
 
-		this.original = o;
-		this.flip_u = false;
-		this.flip_v = false;
+		this.setOriginal( o );
```

Comment:
Yes. I personally consider it less error-prone/easier to simply say "always use setters/getters" than "always check/copy the code of the setter/getter to ensure not missing anything important". Say when a setter is changing two or more mutual fields. It simply one thing less to track.

In terms of performance, setter/getters are usually inlined as they are usually very small.